ludwig/schema/features/preprocessing/image.py

```diff
@@ -109,12 +111,12 @@ class ImagePreprocessingConfig(BasePreprocessingConfig):
         parameter_metadata=FEATURE_METADATA[IMAGE][PREPROCESSING]["infer_image_sample_size"],
     )
 
-    scaling: str = schema_utils.StringOptions(
-        ["pixel_normalization", "pixel_standardization"],
-        default="pixel_normalization",
-        allow_none=False,
-        description="The scaling strategy for pixel values in the image.",
-        parameter_metadata=FEATURE_METADATA[IMAGE][PREPROCESSING]["scaling"],
+    standardize_image: Optional[Union[str, None]] = schema_utils.StringOptions(
```

Comment:
@jimthompson5802 Two things:
1. Just `Optional` or `Union[str, None]` is sufficient. No need to do both ;)
2. When you set `allow_none=True` that will handle adding `None` to the options for you so you don't have to manually add that to the list yourself.

Reply:
Got it. thank you.  Learning something new everyday.